examples/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM rust:1.35 as build
+
+WORKDIR /usr/src/
+
+RUN apt-get update && apt-get install -y \
+    git \
+    cmake \
+    golang \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN git clone --recurse-submodules --depth 1 https://github.com/cloudflare/quiche
+
+RUN cd quiche && cargo build --release --examples
+
+FROM debian:latest
+RUN apt-get update && apt-get install -y \
+    ca-certificates \
+ && rm -rf /var/lib/apt/lists/*
+RUN update-ca-certificates
+WORKDIR /usr/src/
+COPY --from=build /usr/src/quiche/target/release/examples/http3-client ./
+COPY --from=build /usr/src/quiche/target/release/examples/http3-server ./
+COPY --from=build /usr/src/quiche/target/release/examples/client ./
+COPY --from=build /usr/src/quiche/target/release/examples/server ./
+ENV RUST_LOG=trace
```

Comment:
Do we need this by default?

Reply:
I'm open to change this but I'd argue that most people willing to try this today are interested in seeing what is going on with QUIC and HTTP/3. info level hides a lot of detail and obscures some classes of error. I'd like to avoid fielding questions where it seems like the examples are not doing anything because of transient errors.

Reply:
FYI I changed this to `info` and it can be overridden by inquisitive souls.

Reply:
SGTM. Thanks